src/researchhub_document/related_models/feed_exclusion_model.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+from typing_extensions import Required
+
+from researchhub_document.related_models.researchhub_unified_document_model import (
+    ResearchhubUnifiedDocument,
+)
+from utils.models import DefaultModel
+
+
+class FeedExclusion(DefaultModel):
+    # Not a foreign key because hub_id=0 is homepage
+    hub_id = models.IntegerField(
+        default=False,
+        db_index=True,
+    )
+
+    unified_document = models.ForeignKey(
+        ResearchhubUnifiedDocument,
+        default=False,
```

Comment:
I don't think ForeignKeys have a default

Reply:
Apparently they do, I just checked. I think this would just indicate that this field will never have a default value

Reply:
Feels weird to have it included, since it will never have a default value unless we add one in, but it's probably ok.